CIP-xxxx/README.md

```diff
@@ -0,0 +1,136 @@
+---
+CIP: xxxx
+Title: CBOR Tags for Cardano Data Structures and Registry
```

Comment:
```suggestion
Title: CBOR Tags Registry
```
Maybe a cleaner title, I think it can be implied that this is for Cardano Data Structures.

Reply:
Yes @Ryun1 we have to make this change or it will not match the target audience very meaningfully.  The title needs to be kept as clean as possible.  I'll change it in the title & the top level `README.md` file so please @stevenj make it follow suit in the Markdown text... or suggest an concise alternative & we'll follow your suit.